vsts/pipelines/weeklyCleaning.yml

```diff
@@ -0,0 +1,61 @@
+variables:
+  skipComponentGovernanceDetection: true
+
+stages:
+- stage: CleanAgents
+    displayName: Clean OryxLinux Pool
+    jobs:
+    - template: templates/_agentCleanUpJobTemplate.yml
+      parameters:
+        agentName: DDMBLDL004
+      
+    - template: templates/_agentCleanUpJobTemplate.yml
+      parameters:
+        agentName: DDMBLDL005
+    
+    - template: templates/_agentCleanUpJobTemplate.yml
+      parameters:
+        agentName: DDMBLDL006
+
+    - template: templates/_agentCleanUpJobTemplate.yml
+      parameters:
+        agentName: DDMBLDL012
+    - template: templates/_agentCleanUpJobTemplate.yml
+      parameters:
+        agentName: DDMBLDL013
+      
+    - template: templates/_agentCleanUpJobTemplate.yml
+      parameters:
+        agentName: DDMBLDL014
+    
+    - template: templates/_agentCleanUpJobTemplate.yml
+      parameters:
+        agentName: DDMBLDL015
+
+    - template: templates/_agentCleanUpJobTemplate.yml
+      parameters:
+        agentName: DDMBLDL016
+    - template: templates/_agentCleanUpJobTemplate.yml
+      parameters:
+        agentName: DDMBLDL017
+      
+    - template: templates/_agentCleanUpJobTemplate.yml
+      parameters:
+        agentName: DDMBLDL018
+    
+    - template: templates/_agentCleanUpJobTemplate.yml
+      parameters:
+        agentName: DDMBLDL019
+
+    - template: templates/_agentCleanUpJobTemplate.yml
+      parameters:
+        agentName: DDMBLDL020
+    - template: templates/_agentCleanUpJobTemplate.yml
+      parameters:
+        agentName: DDMBLDL021
+      
+    - template: templates/_agentCleanUpJobTemplate.yml
+      parameters:
+        agentName: DDMBLDL022
+
+trigger: none
```

Comment:
Seems like you can add scheduled triggers in yaml itself: https://docs.microsoft.com/en-us/azure/devops/pipelines/build/triggers?view=azure-devops&tabs=yaml#scheduled-triggers

Reply:
as discussed I would do the scheduling using UI